stdlib/2.7/__builtin__.pyi

```diff
@@ -879,3 +880,15 @@ class file(BinaryIO):
     def write(self, data: str) -> None: ...
     def writelines(self, data: Iterable[str]) -> None: ...
     def truncate(self, pos: int = ...) -> int: ...
+
+def apply(object: object, *args: Iterable, **kwargs: Dict[str, Any]) -> Any: ...
```

Comment:
Actually this should probably be something line
```
def apply(func: Callable[[...], _T], args: Sequence[Any] = None, kwds: Mapping[str, Any] = None) -> _T: ...
```
I also strongly advise you to write some test code that uses apply() and play with it in CPython so you know what it does, and then double-check thtat that code is accepted by mypy with these stubs.